pkg/apis/compliance/v1alpha1/complianceremediation_types.go

```diff
@@ -185,6 +191,37 @@ func (r *ComplianceRemediation) HasUnmetDependencies() bool {
 	return (hasDependencies || hasObjDependencies) && !dependenciesMet
 }
 
+func (r *ComplianceRemediation) HasUnsetValues() bool {
+	a := r.GetAnnotations()
+	if len(a) == 0 {
+		return false
+	}
+	_, hasUnsetValues := a[RemediationUnsetValueAnnotation]
+	return hasUnsetValues
+}
+
+func (r *ComplianceRemediation) HasValueRequired() bool {
+
+	a := r.GetAnnotations()
+	if len(a) == 0 {
+		return false
+	}
+	_, hasValueRequired := a[RemediationValueRequiredAnnotation]
+	return hasValueRequired
+
+}
+
+func (r *ComplianceRemediation) HasValueUsed() bool {
+
+	a := r.GetAnnotations()
+	if len(a) == 0 {
+		return false
+	}
+	_, HasValueUsed := a[RemediationValueUsedAnnotation]
+	return HasValueUsed
```

Comment:
There is quite a bit of repetition on these functions now. At this point it would be worth having a common one that takes the annotation to be checked and returns true or false if the annotation is set or not.

Reply:
That's a good advice! So can it be something like this?

`func (r *ComplianceRemediation) HasAnnotation(ann string) bool {

	a := r.GetAnnotations()
	if len(a) == 0 {
		return false
	}
	_, hasAnnotation := a[ann]
	return hasAnnotation

}`

Reply:
looks good